topostats/summary_config.yaml

```diff
@@ -0,0 +1,42 @@
+output_dir: ./output/summary
+csv_file: ./tests/resources/minicircle_default_all_statistics.csv
+file_ext: png
+pickle_plots: True
+var_to_label: null # Optional YAML file that maps variable names to labels, uses topostats/var_to_label.yaml if null
+molecule_id: molecule_number
+image_id: image
+# If both hist and kde are True they are plotted on the same graph, if you only want one you MUST set the other to False
+kde: True
+hist: True
+bins: 20
+stat: count # Aggregate statistic to compute in each bin. Options: count (default), frequency, probability, percent, density
+violin: True
+figsize:
+  - 16
+  - 9
+xrange:
+  - 0
+  - 2
+alpha: 0.5
+palette: bright # Seaborn color palette. Options : colorblind, deep, muted, pastel, bright, dark, Spectral, Set2
+stats_to_sum:  # A list of variables (key) to plot uncomment those that are required.
+  - area
+  - area_cartesian_bbox
+#  - aspect_ratio
+#  - banding_angle
```

Comment:
I cannot see any other references in the codebase to `banding_angle` aside from in `configuration.md`, `summary_config.yaml`, `validation.md`, and `var_to_label.yaml`. Is this statistic yet-to-be implemented?

Reply:
This is a statistic that @Jean-Du has on one of her branches, including it now as she let me know about it already.

Reply:
Thanks for the clarification @Jean-Du 😄

Reply:
Rename to `bending_angle`

Reply:
Corrected spelling.